go.mod

```diff
@@ -5,7 +5,7 @@ go 1.17
 require (
 	github.com/go-logr/logr v1.2.2
 	github.com/go-logr/zapr v1.2.0
-	github.com/openshift/hypershift v0.0.0-20220308210557-1e81ea19568f
+	github.com/openshift/hypershift v0.0.0-20220322200626-7d5d1ee01dd2
```

Comment:
Is this the latest commit for hypershift `release-4.10` branch?

Reply:
The two branches are the same, they are turning off FF this week or early next week.

Reply:
But I re-based with the @release-4.10 to be safe.